### DISCUSSION TITLE
NAV TCAS STBY caution ongoing

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3488] [Help] [answered]
I have the NAV TCAS STBY  caution notification on all the time. I can't remember it not being there. I did a search here but found no mention of it. Can't be normal to fly with this on?

( I posted another one on this topic, I thought, but can't see it. I will delete one if I see it.)

![tcas](https://user-images.githubusercontent.com/75288729/107785313-6bcb6080-6d1a-11eb-96b7-788e652df077.jpg)

**Comment by hotshotp** [comment, marked as answer]
It's your tcas, on the bottom right of the pedestal, set it to the ta/ra section

**Reply by greluc** [reply]
May it be this issue #3490 and this PR #3492?

**Reply by robinottawa** [reply]
Thanks, @greluc. I think it does and I'm testing. I did not think I had to search in PR and Issues separately. I never go in there - way over my head.

**Reply by robinottawa** [reply]
Will test, @hotshotp.

**Reply by robinottawa** [reply]
I can see this is being discussed and worked on, which is great. 
(test results: using latest custom, amber NAV TCAS FAULT notification was displayed over the ADIRS alignment countdown, but went away once aligned. )
